gobblin-service/src/test/java/org/apache/gobblin/service/monitoring/GaaSJobObservabilityProducerTest.java

```diff
@@ -195,6 +197,48 @@ public void testCreateGaaSObservabilityEventWithPartialMetadata() throws Excepti
     serializer.serializeRecord(event);
   }
 
+  @Test
+  public void testCreateGaaSObservabilityFlowEvent() throws Exception {
+    String flowGroup = "testFlowGroup3";
+    String flowName = "testFlowName3";
+    String jobName = JobStatusRetriever.NA_KEY;
+    String flowExecutionId = "1";
+    this.issueRepository.put(
+        TroubleshooterUtils.getContextIdForJob(flowGroup, flowName, flowExecutionId, jobName),
+        createTestIssue("issueSummary", "issueCode", IssueSeverity.INFO)
+    );
+    MockGaaSJobObservabilityEventProducer
+        producer = new MockGaaSJobObservabilityEventProducer(new State(), this.issueRepository, false);
```

Comment:
FSR, I thought that empty `State()` would adopt the default of `false`, which would be to emit a job-schema event for the flow-level (w/ `"NA"` values for some fields).  I thought explicit enablement would be needed to actually convert to and produce a flow-schema event